### DISCUSSION TITLE
How does the board perform measurements?

### DISCUSSION THREAD

**Discussion by vekexasia** [emporia-vue-local/esphome#269] [Q&A]
Hello, I was looking for some info about how the emporia board works under the hood. does anyone have a link or know which components/tricks were used to perform the measurements?

**Comment by flaviut** [comment]
- [Schematic](https://github.com/emporia-vue-local/emporia-vue2-reversing/blob/master/vue2_schematic.pdf)
- [Some schematic discussion](https://github.com/emporia-vue-local/esphome/discussions/59#discussioncomment-4156537)
